### PR TITLE
Mapping/Objects - Car sprites

### PR DESCRIPTION
**Pull request by courierbravo**
Adds objects utilizing the largely unused cars 2-10 sprites. While automobile/random does exist, this will be a nice QoL for mappers. Also, shuffles around the sprites in cars.dmi.

### CODE REVIEW

**Comment by courierbravo** [comment]
!review

File: code/game/objects/structures/urban.dm [hunk 1]
```diff
@@ -20,6 +20,56 @@
 	desc = "A police vehicle with all the bells and whistles you'd expect from a decently-funded agency."
 	icon_state = "copcar"
 
+/obj/structure/automobile/car1
+	name = "blue car"
```
Comment (alsoandanswer) [inline code/game/objects/structures/urban.dm:24]: The naming convention here doesn't match the naming convention of the random car initalize proc, please change it to match the random car initialize proc.

**Comment by courierbravo** [comment]
Requested changes made in an interesting way. I removed the initialize proc, replacing it with a random subtype that spawns one of the cars. I've also made it so each type of sprite has one of the initialize proc's brand names. See .dm for details

**Review by DreamySkrell — APPROVED** [review]

**Comment by courierbravo** [comment]
Accidentally deleted this branch and closed the PR while cleaning up my repo. Smooth brain moment.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**